test/functional/test_framework/test_framework.py

```diff
@@ -154,6 +155,9 @@ def parse_args(self):
                             help="Print out all RPC calls as they are made")
         parser.add_argument("--portseed", dest="port_seed", default=os.getpid(), type=int,
                             help="The seed to use for assigning port numbers (default: current process id)")
+        parser.add_argument("--previous-releases", dest="prev_releases", action="store_true",
+                            default=os.path.isdir(previous_releases_path) and bool(os.path.listdir(previous_releases_path)),
```

Comment:
That's a Python 3.6 syntax  I think
https://docs.python.org/3/library/os.html#os.listdir

Reply:
Nah, just me being stupid. I forgot to test this locally before pushing